docs/src/Playground.tsx

```diff
@@ -5,186 +5,288 @@
+                            name="eventType"
+                            value={eventType}
+                            onChange={(event) => {
+                                setEventType(event.target.value as any)
+                            }}
+                        >
+                            {Object.keys(EventType).map((key) => {
+                                const value = key as EventType
+
+                                return (
+                                    <FormControlLabel
+                                        key={value}
+                                        value={value}
+                                        control={<Radio />}
+                                        label={value}
+                                    />
+                                )
+                            })}
+                        </RadioGroup>
+                    </FormControl>
+                </Grid>
+
+                <Grid item lg={4} md={6} xs={12}>
+                    <FormControl component="fieldset">
+                        <FormLabel component="legend">
+                            Select Symbol Name
+                        </FormLabel>
+                        <RadioGroup
+                            aria-label="symbolName"
+                            name="symbolName"
+                            value={symbolName}
+                            onChange={(event) => {
+                                setSymbolName(event.target.value as any)
+                            }}
+                        >
+                            {SYMBOLS.map((value) => (
+                                <FormControlLabel
+                                    key={value}
+                                    value={value}
+                                    control={<Radio />}
+                                    label={value}
+                                />
+                            ))}
+                        </RadioGroup>
+                    </FormControl>
+                </Grid>
+            </Grid>
+            <H2Component>Example</H2Component>
+            <PreComponent>
+                Example demonstrates how to work with it in <b>React</b>.
+                <br />
+                <br />
+                For debugging you need to look at the console.
+            </PreComponent>
+
+            <Box width="100%" marginTop="12px" marginBottom="12px">
+                <TextField
+                    fullWidth
+                    label="You can provide a list of record fields (separated by comma):"
+                    value={fieldsString}
+                    onChange={(e) => setFieldsString(e.target.value)}
+                    InputProps={{
+                        endAdornment: fieldsString && (
+                            <Button
+                                variant="outlined"
+                                onClick={() =>
+                                    setFields(
```

Comment:
looks like overhead for allow set fields via textedit.

Reply:
What do you suggest?